packages/xstate-react/test/createActorContext.test.tsx

```diff
@@ -122,7 +122,7 @@ describe('createActorContext', () => {
     expect(screen.getByTestId('value').textContent).toBe('b');
   });
 
-  it('should work with useSelector and a custom comparator', async () => {
+  it.skip('should work with useSelector and a custom comparator', async () => {
```

Comment:
Some accidental changes got merged here. This test shouldn't be skipped - nor the `shallowEqual` should be removed from the `useSelector` call

Reply:
I can revert the commit and re-open the PR.

Reply:
Agreed to address separately in https://github.com/statelyai/xstate/pull/3808